ToDo.DataAccess/ToDo.Services/DTOs/TagItemDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace ToDo.Services.DTOs
+{
+    public class TagItemDTO
```

Comment:
You don't need to show this table out of services.

Reply:
Deleted.